deployability/modules/testing/tests/helpers/agent.py

```diff
@@ -355,9 +432,25 @@ def isAgent_port_open(inventory_path):
         """
         os_type = HostInformation.get_os_type(inventory_path)
         if os_type == 'linux':
-            return 'ESTAB' in Executor.execute_command(inventory_path, 'ss -t -a -n | grep ":1514" | grep ESTAB')
+            result = ConnectionManager.execute_commands(inventory_path, 'ss -t -a -n | grep ":1514" | grep ESTAB')
+            if result.get('success'):
+                return 'ESTAB' in result.get('output')
+            else:
+                return False
+
+        elif os_type == 'windows':
+            result = ConnectionManager.execute_commands(inventory_path, 'netstat -ano | Select-String -Pattern "TCP" | Select-String -Pattern "ESTABLISHED" | Select-String -Pattern ":1514"')
+            if result.get('success'):
+                return 'ESTABLISHED' in result.get('output')
+            else:
+                return False
+
         elif os_type == 'macos':
-            return 'ESTABLISHED' in Executor.execute_command(inventory_path, 'netstat -an | grep ".1514 " | grep ESTABLISHED')
+            result = ConnectionManager.execute_commands(inventory_path, 'netstat -an | grep ".1514 " | grep ESTABLISHED')
+            if result.get('success'):
+                return 'ESTABLISHED' in result.get('output')
+            else:
+                return False
 
     def get_agents_information(wazuh_api: WazuhAPI) -> list:
```

Comment:
Decorate this method as `@staticmethod`

Reply:
This method depends on the WazuhApi instantiation so the decorator could alter its operation